packages/backend/src/api/controllers/liveness/LivenessController.ts

```diff
@@ -39,6 +40,34 @@ export class LivenessController {
         const withAnomalies = calculateAnomaliesPerProject(intervals)
         console.timeEnd(`withAnomalies ${project.projectId.toString()}`)
 
+        if (withAnomalies) {
+          const l2project = layer2s.find((l) => l.id === project.projectId)
```

Comment:
get rid of this find, add duplicateData optional field to backend liveness config